src/main/resources/OH-INF/thing/controller_serial.xml

```diff
@@ -72,6 +72,13 @@
                 <advanced>true</advanced>
             </parameter>
 
+            <parameter name="controller_maxbatterywakeduration" type="integer" groupName="network" min="5" max="20">
```

Comment:
I'd suggest to make this less that 5 seconds - many people like it to be a lot less (this was a very hot topic a few years ago which is why I had to trim it down). I'd suggest 2 seconds.

Reply:
I lowered the minimum to 2, but it will blow up on anyone that tries it, especially if we can't keep the add Node serial message changes. The sleep message will get queued before the device can do ANY configuration
          `  <parameter name="controller_maxawakeperiod" type="integer" groupName="network" min="2" max="20">`